Src/WitsmlExplorer.Frontend/components/Modals/UserCredentialsModal.tsx

```diff
@@ -89,12 +53,15 @@ const UserCredentialsModal = (props: UserCredentialsModalProps): React.ReactElem
       password
     };
     try {
-      const encryptedPassword = await CredentialsService.verifyCredentials(credentials);
-      props.onConnectionVerified({ ...credentials, password: encryptedPassword });
```

Comment:
Not calling props.onConnectionVerified breaks every use of CredentialsMode.TEST

Reply:
ok, i'll reinstate it